settings.json.sample

```diff
@@ -130,6 +130,15 @@
       "webhook_url": "",
       "rule_list": ["custom_keywords"]
     },
+    "http_output": {
+      "enabled": false,
+      "module": "outputs.http_output",
+      "classname": "HttpOutput",
+      "endpoint_url": "",
+      "http_auth": false,
```

Comment:
I think a more elegant solution would be setting this up to be a headers dictionary, where you could set stuff like basic auth if you wanted to, but could also use this output for other outputs (such as a splunk HEC or a webhook).

Reply:
I added an array of headers in the settings, but I think it is useful to leave the username / password options for auth basics, which avoids preparing the header